src/views/MainView/index.tsx

```diff
@@ -1,8 +1,8 @@
 import { Box } from '@chakra-ui/react'
 import { MDXRemoteSerializeResult } from 'next-mdx-remote'
-import Link from 'next/link'
-import { useRouter } from 'next/router'
 
+import { useRouter } from 'next/router'
+import { CardGroup } from '@components'
 import { Markdown } from '@components'
```

Comment:
Somehow husky scripts don't work with you. Otherwise we wouldn't see this kind of linting/formatting issues.

Can you please run `npm run lint:fix` before pushing your changes until we fix this issue.